docs/iris/src/whatsnew/2.2.rst

```diff
@@ -25,11 +25,78 @@ Iris 2.2.0 Features
   contiguous, or the cube's data must be masked at the discontiguities in
   order to avoid plotting errors.
 
-  The iris plot functions :func:`iris.plot.quiver` has been added, and this
+  The iris plot function :func:`iris.plot.quiver` has been added, and this
```

Comment:
iris --> Iris please (Iris is a proper noun). (Also elsewhere within the document, possibly outside of the changes proposed here.)